### DISCUSSION TITLE
Element coordinates depend on I_solid

### DISCUSSION THREAD

**Discussion by victorkemp** [OpenRadioss/OpenRadioss#2905] [Q&A]
I've noticed that the brick element local coordinates described in the users guide only seem to work for parts with I_solid = 1, 14, or 24. If you set it to 2, 17, or 18 (with corotational turned on where it's optional), the stress in the output has the axes rotated so that where it "should" be x,y,z, it's z,x,y or maybe y,z,x. Is that how it's supposed to be and is there some other factor that determines it? Those I_solid options don't seem to have anything in common that suggest different element coordinates.

**Comment by qiangRadioss** [comment]
Hi Victor,

Isolid=14,24,18 use anyway co-rotational formulation w/o look at Iframe value, the optional case w/ Iframe works w/ Isolid=1,2,17, for the output (stress,strain...), they are generally in the basic (global) system then don't need to know which Isolid value used. that's at least true for Anim&H3d (don't know which kind of animation file you used), and for time history output (T file) you have optional output both in basic or local coordinate system. 

hope this helps.

Qiang.

**Reply by victorkemp** [reply]
I'm inferring the coordinate system from brick stress in the STY output so it might just be a STY issue.
